java-frontend/src/test/java/org/sonar/java/se/checks/FlowComputationTest.java

```diff
@@ -72,6 +71,10 @@ public void test_multiple_paths() {
   @Test
   public void test_multiple_paths_xproc() {
     JavaCheckVerifier.verify("src/test/files/se/FlowComputationMultiplePathXProc.java", new NullDereferenceCheck(), new ConditionAlwaysTrueOrFalseCheck());
+  }
 
+  @Test
+  public void test_trigger_yield_flow_computation_only_on_relevant_yields() throws Exception {
+    JavaCheckVerifier.verify("src/test/files/se/UselessFlowComputation.java",  new NullDereferenceCheck(), new ConditionAlwaysTrueOrFalseCheck());
```

Comment:
This test is annoying me. If I understand what it is indirectly testing, we provide no way to verify that no flow has been computed for `fun2` and `fun3`. If we change this behavior, and restart computing flow for all the methods, this test won't fail. Drop it?